jenkins-jobs/docker/debezium-testing-system/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM registry.access.redhat.com/ubi8/openjdk-17
-USER root
-RUN microdnf install git   
-RUN microdnf install unzip
+FROM fedora:35 AS base
 
-ARG OCP_CLIENT=3.7.23
+RUN dnf -y install git unzip zip findutils
 
+ARG OCP_CLIENT=3.7.23
```

Comment:
OCP 3.x is quite outdated. We want ideally the latest 4.x 

https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/latest/openshift-client-linux.tar.gz


is the one you want. This particular URL also targets the latest 4.x client